src/vivarium_cluster_tools/vipin/perf_report.py

```diff
@@ -210,6 +293,8 @@ def report_performance(
     # Set index to include branch configuration/scenario columns
     perf_df, scenario_cols = set_index_scenario_cols(perf_df)
 
+    append_perf_data_to_central_logs(perf_df, output_directory)
```

Comment:
I think we'll want a try-except block actually. Where would we want to put this? Here, or within the function itself?

Reply:
if you want to try the entire function, I'd wrap it here as opposed to internally and then having to indent everything.

(and I agree we should try/except this thing at least until we get a handle on how stable it is. The question then is what do we do if this portion fails? Do we go back and try and add the failed logs to the main log or just fix the issue and let it go?)